http_handlers.go

```diff
@@ -34,13 +35,23 @@ func webhook(c *gin.Context) {
 		clog.Infof("Alerts: GroupLabels=%v, CommonLabels=%v", m.GroupLabels, m.CommonLabels)
 
 		for _, alert := range m.Alerts {
+			alertName := alert.Labels["alertname"]
 			// shorten all alert annotation URLs
 			cli := NewLinksClient()
 			for k, txt := range alert.Annotations {
-				err, n := cli.ReplaceLinks(c, txt)
+				err, n := cli.ReplaceLinks(ctx, txt)
 				if err != nil {
 					err = errors.Wrap(err, "Error shortening one or more links")
```

Comment:
This is perhaps a bit ambiguous as it's only ever going to report the first failure. Not quite sure how I'd word it either mind.

Reply:
maybe something like 
```suggestion
					err = errors.Wrap(err, "Error whilst shortening links")
```

Reply:
Sounds good to me.